### ISSUE TITLE
[SI-ERROR-TRUE] 1.1.1 Non-text Content - Decorative images (Issue Content Type)

### ISSUE THREAD

**Issue by cathysnider** [CuBoulder/express#1033]
## Problem Description per Siteimprove

This image has an empty alt text, which indicates that it only has a decorative function. If that is the case the page is in compliance with this criterion.

## Page Description

An issue, built with the Issue Content Type.  

## What we're doing

The Issue Cover Image does not have an alt tag, or a field to add an alt tag. 

## Solution

Possibly have the Issue Page Title become the alt tag of the Image Cover. Or add a field so that the Issue Cover Image can be described more completely.

Example: https://www-test.colorado.edu/p1386f7a359a/
